### DISCUSSION TITLE
Thinkathon 2021 Guidelines

### DISCUSSION THREAD

**Discussion by emilyacook** [mattermost/community#5] [Announcements]
Deadline for prizes: June 25 @ 11:59 PM PST
Deadline for submissions: never!

# What is a Thinkathon?

Have you ever been working in Mattermost and thought to yourself, "Gee, wouldn't it be cool if we had [XYZ]." Or "[XYZ] would make my experience so much better"? I'm sure many of us have, but what do you do with those ideas? Unless you're sure they're feasible and worthwhile to implement, you probably don't open a [Feature Proposal](https://mattermost.org/feature-ideas/).  _Maybe_ you throw it out in a meeting, or wait until a Hackathon to try and propose it. But generally, most of these ideas just stay in your head, never to be revealed.

Well, we want to give you space to reveal them. The theme of this MatterCon is _New Ideas_, and we want to give you not only the time to think about new ideas, but also a dedicated, open place to share them and discuss.

# How do I submit a discussion item?

Go to `Discussions` -> `New discussion`

<img width="1658" alt="Screen Shot 2021-06-22 at 10 08 40 AM" src="https://user-images.githubusercontent.com/35075700/122950542-265f1580-d342-11eb-9a33-19a38eb6c114.png">

Write in a title and description, select a category, and hit `Start discussion`

<img width="1495" alt="Screen Shot 2021-06-22 at 10 09 57 AM" src="https://user-images.githubusercontent.com/35075700/122950671-4262b700-d342-11eb-908c-f092021f46e1.png">

Then you're done!

# Are there rules to what can be posted?

No! As long as you're following the [Contributor Code of Conduct](https://handbook.mattermost.com/contributors/contributors/contribution-guidelines#contributor-covenant-code-of-conduct). Other than that, anything goes. You can post brand new ideas, completely crazy ideas, ideas that may already have an issue open, something you've been talking about for years and years, even company or community improvement ideas.

Also, just to be completely clear, *you do not have to be Mattermost staff to participate.* This is open to everyone!

# Are there other rules?

Yes, there are two.

### Rule 1: Be kind

Ideas may be completely out of the realm of possibility. This is a safe place for those to be posted. Please help maintain that atmosphere. You can give insights as to why something may not be feasible, but *do not* shoot down anyone else's ideas.

Basically, we want this to be as open as possible, so please do not make people feel bad or dumb for an idea -- if you don't like something, just move on to the next one.

There will be 0 tolerance for this to ensure everyone feels safe posting whatever they want. I (@emilyacook) will give you a warning the first time you're in violation, any violations after that will disqualify you from participation.

### Rule 2: Have no expectation of implementation

The flip-side to this being a completely open idea forum is that *there is no guarantee that anything posted will be implemented*. Even if something is the #1 upvoted idea.

**We will be taking the submissions from here and holding a Hackathon from it**, so please still be as detailed as possible in case anyone does decide to pick it up and implement it. Focalboard started as a hackathon idea, so yours might too. But it also might not. Be prepared for that.

# What are the categories?

**Moonshot ideas**: Innovative, big ideas!
**Practical ideas**: basically anything that could be reasonably implemented and add value
**Enhancement ideas**: how to make an existing feature more useful
**Fun ideas**: things that would make Mattermost more fun
**General ideas**: anything else, or where people can post if they don't know how to categorize further

Completely accurate categorization is not required, it's just a useful way for people to browse different types of ideas. If you're not sure your idea is in the right category, don't worry about it! One of the organizers may re-categorize it, but they're more to serve as over-arching idea themes than strict guidelines.

# Are there prizes?

Yes! We will be giving out 5 pairs of exclusive pairs of Mattermost branded AirPod Pros for the following:
- Top 3 most active participants (including submissions, commenting, and upvoting)
- Most upvoted submission
- Nomination: at the end, everyone will be able to nominate one person (excluding themselves). The one with the most nominations wins.

Notes:
- In the event of a tie, we will be [choosing at random](https://www.random.org/)
- If one person wins multiple categories, we will be selecting the next highest person
   - i.e. you can only win one pair, sorry!

![](https://aws1.discourse-cdn.com/business7/uploads/mattermost/original/2X/9/999bcb443df689a81795e42a6a99aefb414b4927.jpeg)

# Where should I leave feedback for this event?

For any feedback, questions, comments, concerns, whatever, please leave them in the comments of this post :)

**Comment by esethna** [comment]
This is such a great idea @emilyacook, stoked to see all the ideas circulating, thanks for organizing!

**Comment by chetanyakan** [comment]
What is the timeline for this?

**Reply by emilyacook** [reply]
@chetanyakan Deadline to win prizes is the end of MatterCon (12:00AM PST on June 25th), but you can continue submitting as long as you want :) (just wont win prizes)

**Reply by emilyacook** [reply]
Actually @chetanyakan I'm going to extend it one day, to give some of the newer submissions a chance to pick up some discussion :)

**Reply by cnu1812** [reply]
@emilyacook Awesome
